### DISCUSSION TITLE
add the api's interface address

### DISCUSSION THREAD

**Discussion by Wenzaiyi** [ricklamers/gpt-code-ui#50] [Q&A]
I'd like to be able to add the api's interface address to the sidebar, also, it would be great if it supported the download of Excel tables!

**Comment by ricklamers** [comment]
![image](https://github.com/ricklamers/gpt-code-ui/assets/1309307/2f9b83fc-65be-4cfc-9977-d7cbfa88545b)

The file download should be fixed now.

API to the interface is out of scope for the project. It's not meant to be used as an API service, although everyone can do so if they really want to :)
